src/components/ResetPassword/RequestResetLink.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react';
+import ImageFader from '../Common/ImageFader';
+import { defineMessages, useIntl } from 'react-intl';
+import LanguagePicker from '../Layout/LanguagePicker';
+import Button from '../Common/Button';
+import { Field, Form, Formik } from 'formik';
+import * as Yup from 'yup';
+import axios from 'axios';
+
+const messages = defineMessages({
+  forgotpassword: 'Forgot your password?',
+  emailresetlink: 'Email me a Recovery Link',
+  email: 'Email',
+  validationemailrequired: 'Must be a valid email address',
+  gobacklogin: 'Go Back to Login Page',
+  successmessage:
+    'If the email is connected to a user then a link to reset the password has been sent.',
```

Comment:
How about: "A password reset link will be sent to the provided email address if it is associated with a valid user."